docs/how_to/cluster_hard_way.md

```diff
@@ -156,6 +156,8 @@ Shortly after, you should see your node complete bootstrapping:
 20:10:14.764771[I] successfully updated topology to 3 hosts
 ```
 
+If you need to setup multiple namespaces, you can run the above `/api/v1/database/create` command multiple times with different namespace configurations.
```

Comment:
There is a special one. I think its `/api/v1/database/namespace/create`

Reply:
Really? I thought you could use the database create api to create an additional namespace? @robskillington

Reply:
Yeah they are the same endpoint:
```
	// Register the same handler under two different endpoints. This just makes explaining things in
	// our documentation easier so we can separate out concepts, but share the underlying code.
	r.HandleFunc(CreateURL, wrapped(createHandler).ServeHTTP).Methods(CreateHTTPMethod)
	r.HandleFunc(CreateNamespaceURL, wrapped(createHandler).ServeHTTP).Methods(CreateNamespaceHTTPMethod)
```

Maybe we should just mention one though to make it less confusing to end users.